File: patch_bots.js
const fs = require('fs');
const botsPath = require('os').homedir() + '/.botfleet/bots.json';
const bots = JSON.parse(fs.readFileSync(botsPath, 'utf8'));

let changed = false;
for (const [id, bot] of Object.entries(bots)) {
  if (bot.modelSelection && bot.modelSelection.instanceId === 'codex') {
    bot.modelSelection.instanceId = 'antigravity';
    bot.modelSelection.model = 'gemini-2.5-pro'; // Or any valid model, they can change it later
    changed = true;
  }
}

if (changed) {
  fs.writeFileSync(botsPath, JSON.stringify(bots, null, 2));
  console.log('Patched bots.json');
}

File: patch_bots.py
import json, os

bots_path = os.path.expanduser('~/.botfleet/bots.json')
with open(bots_path, 'r') as f:
    bots = json.load(f)

changed = False
for bot in bots:
    if bot.get('modelSelection', {}).get('instanceId') == 'codex':
        bot['modelSelection']['instanceId'] = 'antigravity'
        bot['modelSelection']['model'] = 'gemini-2.5-pro'
        bot['lastInstanceId'] = 'antigravity'
        changed = True

if changed:
    with open(bots_path, 'w') as f:
        json.dump(bots, f, indent=2)
    print("Patched")
